JSON/Pixel Number Base.json

```diff
@@ -1,19 +1,21 @@
 {
-  "Compatibility": "Untested",
+  "Author": "Panoptes (Xel), BigCrunch22",
```

Comment:
Author field is not required if Contributor field is present. This applies to all other jsons in case it went through.

Reply:
Fixed.